common/src/main/scala/org/apache/celeborn/common/exception/CelebornServiceException.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.common.exception
+
+class CelebornServiceException(message: String, cause: Throwable)
```

Comment:
Is it possible to reuse CelebornException instead of introducing a new one?

Reply:
No, as CelebornException is a checked exception, then it would make every called method should change their signature, use RuntimeException such as CelebornServiceException would be better.

Reply:
> 

Is it possible to rename to CelebornRuntimeException? CelebornServiceException is not quite straitforward.

Reply:
> > 
> 
> Is it possible to rename to CelebornRuntimeException? CelebornServiceException is not quite straitforward.

OK